src/i18n/i18n.ts

```diff
@@ -0,0 +1,32 @@
+import i18n from 'i18next';
+import {initReactI18next} from 'react-i18next';
+import LanguageDetector from "i18next-browser-languagedetector";
+import en from './locales/en-GB/translation.json';
+import fr from './locales/fr-FR/translation.json';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+i18n.use(initReactI18next)
+    .use(LanguageDetector)
+    .init({
```

Comment:
do we want to set up the default language (which should probably be french as it has the broadest French community) ?

Reply:
That's what I had initially but I think the LanguageDetector should take care of that, as most browsers will be supported. Happy to set it as default though.
My laptop and VMs are all in English, so I have to confess I didn't try. What happens when you run it?